pylint/checkers/method_args.py

```diff
@@ -41,7 +41,6 @@ class MethodArgsChecker(BaseChecker):
             "positional-only-arguments-expected",
             "Emitted when positional-only arguments have been passed as keyword arguments. "
             "Remove the keywords for the affected arguments in the function call.",
-            {"minversion": (3, 8)},
```

Comment:
Shouldn't we replace this with a py-version check ?

Reply:
Hmm, but if my library supports 3.7 - 3.12, and I'm using a 3.8 function with positional-only arguments in the wrong manner, wouldn't I rather continue to see the message even if `--py-version=3.7`, because I certainly have bigger problems anyway if I'm using a 3.8+ function when I supposedly support 3.7?

Reply:
I thought `--py-version` was most helpful for ensuring we silence suggestion-style messages that don't make sense on lower versions, like `consider-using-f-string`, but here we have a message that won't ever get raised on a lower version anyway, so I don't think we need to add a `--py-version` workaround for it.

Reply:
I agree with Jacob's second comment. I don't think it will add much benefit to spend the time to add support for that edge case. Removing this line is fine.